powertools-parameters/src/test/java/software/amazon/lambda/powertools/parameters/ParamManagerIntegrationTest.java

```diff
@@ -116,4 +120,16 @@ public void secretsProvider_get() {
         assertThat(secretsProvider.get("keys")).isEqualTo(expectedValue); // second time is from cache
         verify(secretsManagerClient, times(1)).getSecretValue(any(GetSecretValueRequest.class));
     }
+
+    @Test
+    public void getDynamoDbProvider() {
```

Comment:
Here I simply test we can get a provider through the `ParamManager`. Usage of the provider is tested in `DynamoDbProviderTest`.